### PR TITLE
Update Gravatar's sign-up and log-in pages

### PR DESCRIPTION
**Pull request by wellyshen**
Related to 105072-gh-Automattic/gravatar

## Proposed Changes

To tweak Gravatar's sign-up and log-in pages (p8Slzc-e4-p2#comment-418). Changes are as follows:

* To add a "Continue with WordPress.com" option in the social buttons area of the sign-up page
* To adjust the title and subtitle of the sign-up and log-in pages
* To add a "Gravatar is powered by WordPress.com" footer for the sign-up and log-in pages

## Testing Instructions

* Switch to the branch and run it locally (check [the doc](https://github.com/Automattic/wp-calypso#getting-started) to setup your dev environment)
* Log out of your WP account
* Go to the log-in page by [this URL](http://calypso.localhost:3000/log-in?client_id=1854&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3Dad4b462bedc61ff0ee2b72b16252632bcf4f4ec2e9aaab25a169e49d565e87bf%26redirect_uri%3Dhttps%253A%252F%252Fen.gravatar.com%252Fconnect%252F%253Faction%253Drequest_access_token%26from-calypso%3D1) (you might need to modify your origin), and you will see the new Gravatar's log-in page as below:

| Before | After |
| - | - |
| login_1 | login |

* Go to the sign-up page by [this URL](http://calypso.localhost:3000/start/wpcc/oauth2-user?ref=oauth2&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3D4c419247334b07b3cb7adc7fbdfe0a723abe7296d6e38f19146f5353a3d88e69%26redirect_uri%3Dhttps%253A%252F%252Fen.gravatar.com%252Fconnect%252F%253Faction%253Drequest_access_token%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login%26from-calypso%3D1&oauth2_client_id=1854) (you might need to modify your origin), and you will see the new Gravatar's sign-up page as below:

| Before | After |
| - | - |
| signup_1 | signup |

* Click the "Continue to WordPress.com" and you will be redirected to Gravatar's log-in page

* The changes don't influence the log-in page of other services (please run Calypso locally for dev):
  * WordPress:
    * [Production](https://wordpress.com/log-in/)
    * [Development](http://calypso.localhost:3000/log-in)
  * WooCommerce:
    * [Production](https://wordpress.com/log-in?client_id=50916&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D89c377eff8f6cc5e58dbfdd2138e883f2f75933aa7d5876347291891955cdcae%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26from-calypso%3D1)
    * [Development](http://calypso.localhost:3000/log-in?client_id=50916&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D89c377eff8f6cc5e58dbfdd2138e883f2f75933aa7d5876347291891955cdcae%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26from-calypso%3D1)
  * Jetpack:
    * [Production](https://wordpress.com/log-in?client_id=69040&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252Fpricing%26scope%3Dglobal%26blog_id%3D0%26from-calypso%3D1)
    * [Development](http://calypso.localhost:3000/log-in?client_id=69040&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252Fpricing%26scope%3Dglobal%26blog_id%3D0%26from-calypso%3D1)
  * Akismet:
    * [Production](https://wordpress.com/log-in?client_id=973&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D973%26response_type%3Dcode%26blog_id%3D0%26state%3D202e9384cd96c958a22d138bf35bf102662bd4dee8357efe8d2207f7450e3d80%26redirect_uri%3Dhttps%253A%252F%252Fakismet.com%252Faccount%252F%253Faction%253Drequest_access_token%26implicit%3Dtrue%26new-user%3D0%26from-calypso%3D1)
    * [Development](https://wordpress.com/log-in?client_id=973&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D973%26response_type%3Dcode%26blog_id%3D0%26state%3D202e9384cd96c958a22d138bf35bf102662bd4dee8357efe8d2207f7450e3d80%26redirect_uri%3Dhttps%253A%252F%252Fakismet.com%252Faccount%252F%253Faction%253Drequest_access_token%26implicit%3Dtrue%26new-user%3D0%26from-calypso%3D1)
  * Crowdsignal:
    * [Production](https://wordpress.com/log-in?client_id=978&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D710b3271d959aaa6a02c3be9e799038443511289283f9608b77d35ea5a5b3659%26redirect_uri%3Dhttps%253A%252F%252Fapp.crowdsignal.com%252Fconnect%253Fsource%253Dlogin%2526action%253Drequest_access_token%26wpcom_connect%3D1%26from-calypso%3D1)
    * [Development](http://calypso.localhost:3000/log-in?client_id=978&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D710b3271d959aaa6a02c3be9e799038443511289283f9608b77d35ea5a5b3659%26redirect_uri%3Dhttps%253A%252F%252Fapp.crowdsignal.com%252Fconnect%253Fsource%253Dlogin%2526action%253Drequest_access_token%26wpcom_connect%3D1%26from-calypso%3D1)
  * P2
    * [Production](https://wordpress.com/log-in?redirect_to=https%3A%2F%2Fwordpress.com%2Fstart%2Fp2%2Fp2-confirm-email%3Fref%3Dp2-lp&from=p2&signup_url=%2Fstart%2Fp2%2Fuser%3Fref%3Dp2-lp)
    * [Development](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fwordpress.com%2Fstart%2Fp2%2Fp2-confirm-email%3Fref%3Dp2-lp&from=p2&signup_url=%2Fstart%2Fp2%2Fuser%3Fref%3Dp2-lp)

* The changes don't influence the sign-up page of other services (please run Calypso locally for dev):
  * WordPress:
    * [Production](https://wordpress.com/start/user)
    * [Development](http://calypso.localhost:3000/start/user)
  * WooCommerce:
    * [Production](https://wordpress.com/start/wpcc/oauth2-user?oauth2_client_id=50916&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Daea4facd53745314427e505c0284d85f9294c49fc82139da9a1e10fdf469bdf1%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26from-calypso%3D1)
    * [Development](http://calypso.localhost:3000/start/wpcc/oauth2-user?oauth2_client_id=50916&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Daea4facd53745314427e505c0284d85f9294c49fc82139da9a1e10fdf469bdf1%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26from-calypso%3D1)
  * Jetpack:
    * [Production](https://wordpress.com/start/wpcc/oauth2-user?oauth2_client_id=69040&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0%26from-calypso%3D1)
    * [Development](http://calypso.localhost:3000/start/wpcc/oauth2-user?oauth2_client_id=69040&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0%26from-calypso%3D1)
  * Crowdsignal:
    * [Production](https://wordpress.com/start/crowdsignal/oauth2-name?oauth2_client_id=978&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D8c8d8cb0778330a9ada158ee7ef5d6ed8c93306215f5bc54456fa2e0bfadf220%26redirect_uri%3Dhttps%253A%252F%252Fapp.crowdsignal.com%252Fconnect%253Fsource%253Dlogin%2526action%253Drequest_access_token%26wpcom_connect%3D1%26from-calypso%3D1&user_email=wellyshen)
    * [Development](http://calypso.localhost:3000/start/crowdsignal/oauth2-name?oauth2_client_id=978&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D8c8d8cb0778330a9ada158ee7ef5d6ed8c93306215f5bc54456fa2e0bfadf220%26redirect_uri%3Dhttps%253A%252F%252Fapp.crowdsignal.com%252Fconnect%253Fsource%253Dlogin%2526action%253Drequest_access_token%26wpcom_connect%3D1%26from-calypso%3D1&user_email=wellyshen)
  * P2
    * [Production](https://wordpress.com/start/p2/user?ref=p2-lp&user_email=wellyshen)
    * [Development](http://calypso.localhost:3000/start/p2/user?ref=p2-lp&user_email=wellyshen)

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by digitalwaveride** [comment]
Welly, thanks so much for working on this!!! 🙏 

I think it's a great improvement, reduces the noise and makes the site much simpler 👍

A few little things I noticed:

**- A "log in" link is missing**
Of course, users could use the "Continue with WordPress.com" path, but some users might not remember or get the context that Gravatar is based on WP.com accounts etc... especially if they are actively looking for something that says "login" which is standard on most other signup pages.
Three potential options:
-> 1) We add the link similar to the other views in the subline below the headline, with a copy like this: "Do you have already an account? Log in"
-> 2) We add a "Log in" link to the top right corner of the screen
-> 3) We write instead of "Continue with WordPress.com" -> "Log in with WordPress.com"

For consistency reasons with other WP.com views, I would prefer option 1).

**- Font-size of the legal texts are different**
I noticed that the same is the case currently on the signup/login views on WP.com, but I am wondering if that might be some kind of a glitch? Having them both the same font-size, 12px or 14px (if legally required), would make things much calmer.

**- It might make sense to checkin with legal if the copy changes of the site are ok**, or if there are any concerns or needs to make it more obvious what kind of account will be created -> a WordPress.com account.


**- Visually I briefly experimented if it might look better to align/center the three logo circles with the "OR" ...** 

image

... but I was in the end not convinced if it makes things really better.

**Comment by wellyshen** [comment]
@digitalwaveride Thank you for the feedback, these are really helpful, will adjust the PR 💯

**Comment by digitalwaveride** [comment]
Thanks so much, Welly! 

Sign up view is looking great 😻 !

Two tiny notes:
- Regarding the "Powered by" line at the bottom, I would not separate the WordPress logo from the brand name "WordPress.com", but keeping the logo in front of WordPress.com

image


- On the responsive views there is one state, see screenshot, at starting 780px where the "logo circles" are jumping out of alignment.

image


Otherwise, all good!!!

---

Regarding the log in view:

@burtrw made an interesting headline suggestion [here](p8Slzc-e4-p2#comment-427) 

Besides that, looking at the copy, and imagining that a user has been redirected here without a WordPress context, it might be not clear to most users, which kind of "username/email" they should use here. Mentioning WordPress.com could help... maybe via a subline, similar to the styling we are using in the signup view?

Combined with Ronnie's headline... it could look something like this?

image

What do you think?

**Comment by wellyshen** [comment]
@digitalwaveride I have addressed all the feedback, please [take a look](https://github.com/Automattic/wp-calypso/pull/76531#issue-1693574974), thanks!

**Review by digitalwaveride — APPROVED** [review]
Thanks so much Welly for all the adjustments! 🙏🙏🙏

From the design side of things 👍  ... I think we are good to go.

File: client/blocks/signup-form/social.jsx [hunk 1]
```diff
@@ -95,6 +97,17 @@ class SocialSignupForm extends Component {
 		}
 	};
 
+	renderWordPressLoginLink() {
+		return (
+			<a className="social-buttons__button button" href={ this.props.loginUrl }>
```
Comment (lamosty) [inline client/blocks/signup-form/social.jsx:102]: Should we follow the `signup-form__` prefix?

**Review by lamosty — APPROVED** [review]
Code looks good, left a small nit.

LGTM! :shipit:

File: client/blocks/signup-form/social.jsx (hunk 1 above)
Comment (wellyshen) [reply client/blocks/signup-form/social.jsx:102]: That's fine, coz we just re-use the existing classes

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7979627

Thank you @wellyshen for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7979627).

**Pull request merged**